plenum/test/view_change_with_delays/test_delayed_instance_changes.py

```diff
@@ -17,14 +17,12 @@ def check_vcd_msgs(node, expected_count, expected_view_no):
     assert expected_count == vcd_count
 
 
-def check_no_ic_msgs(node, expected_view_no):
-    if expected_view_no not in node.msgsForFutureViews:
-        raise AssertionError('no messages for expected view')
-    ic_count = sum(1 for msg in node.msgsForFutureViews[expected_view_no] if isinstance(msg[0], InstanceChange))
-    assert ic_count == 0
+def check_no_ic_msgs(node, expected_view_no, nodes):
+    assert node.view_changer.instance_changes.has_view(expected_view_no)
+    for n in nodes:
+        assert not node.view_changer.instance_changes.has_inst_chng_from(expected_view_no, n)
 
 
-@pytest.mark.skip(reason="INDY-2223: Temporary skipped to create build")
 def test_delayed_instance_changes_after_vcd_for_next_view(looper, txnPoolNodeSet):
```

Comment:
I think it would be great to call `ensure_pool_functional` to make sure that all nodes have equal data and the pool can order at the end of every test.